pages/common/gh.md

```diff
@@ -3,34 +3,34 @@
 > Work seamlessly with GitHub from the command line.
 > More information: <https://cli.github.com/>.
 
-- Create a pull request:
+- Clone a repository locally:
 
-`gh pr create`
+`gh repo clone {{owner}}/{{repository}}`
 
-- View a pull request in the browser:
+- Create a new issue:
 
-`gh pr view {{pr_number}}`
+`gh issue create`
 
-- Check out pull requests locally:
+- View and filter repository’s open issues:
 
-`gh pr checkout {{pr_number}}`
+`gh issue list`
 
-- Check the status of your pull requests:
+- View an issue in the browser:
 
-`gh pr status`
+`gh issue view --web {{issue_number}}`
 
-- Clone a repository locally:
+- Create a pull request:
 
-`gh repo clone {{owner}}/{{repository}}`
+`gh pr create`
 
-- Create a new issue:
+- View a pull request in the browser:
 
-`gh issue create`
+`gh pr view --web {{pr_number}}`
 
-- View and filter repository’s open issues:
+- Check out pull requests locally:
 
-`gh issue list`
+`gh pr checkout {{pr_number}}`
 
-- View an issue in the browser:
+- Check the status of your pull requests:
```

Comment:
Generally we try to avoid the 2nd person (e.g. `you`, `your`, etc) in tldr-pages :slightly_smiling_face:

Can this be reworded at all?